FirebaseAppCheck/Sources/Core/FIRAppCheckLogger.m

```diff
@@ -16,6 +16,20 @@
 
 #import "FirebaseAppCheck/Sources/Core/FIRAppCheckLogger.h"
 
+#import "FirebaseCore/Sources/Private/FirebaseCoreInternal.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
 FIRLoggerService kFIRLoggerAppCheck = @"[Firebase/AppCheck]";
 
 NSString *const kFIRLoggerAppCheckMessageCodeUnknown = @"I-FAA001001";
+
+void FIRAppCheckDebugLog(NSString *message, ...) {
+  va_list args_ptr;
+  va_start(args_ptr, message);
+  FIRLogBasic(FIRLoggerLevelDebug, kFIRLoggerAppCheck, kFIRLoggerAppCheckMessageCodeUnknown,
```

Comment:
Nit: If this is landing any time soon, I'd prefer that we pass in the code number inside the function and then pass in `Unknown` everywhere so it's easier to refactor at a later date.  Just want to make sure we don't accidentally ship multiple logs with the same Unknown error code.

Reply:
Actually, I intentionally left the default message code here. I don't remember when I actually used the code last time, so it looks like it's has very limited usefulness. I would prefer to keep track of the codes only for logs with higher level and use the default one for debug. WDYT?

Reply:
The codes are more for external usage - if  a developer runs into a log statement they can then just search for that code to either find the code itself or other posts referencing that error message, for example:

https://www.google.com/search?q=%22I-COR000003%22

That way if we need to wordsmith the error message at all we can do so and not lose any historical context.

Reply:
Ah, I didn't think of this particular use case. I'll add codes. Though it still looks much more relevant for non-debug logs, it may be useful for debug as well.

Reply:
updated codes here and in other log messages.